.gitmodules

```diff
@@ -10,3 +10,6 @@
 	path = micro-ros-stm32_embeddedrtps/micro-ros_embeddedrtps/micro_ros_stm32cubemx_utils
 	url = https://github.com/mROS-base/micro_ros_stm32cubemx_utils
 	branch = galactic
+[submodule "mros2-posix"]
+	path = mros2-posix
+	url = git@github.com:mROS-base/mros2-posix.git
```

Comment:
It's better to denote `branch` or `tag` to clarify the version for evaluation. Please specify it.

Reply:
Done: https://github.com/mROS-base/eval/pull/3/commits/25949ca05f259dd90840e8f82fa978d152fc79b4